mutiny-wasm/src/lib.rs

```diff
@@ -38,8 +37,8 @@ pub struct NodeManager {
 /// It can be configured to use all different custom backend services, or to use the default
 /// services provided by Mutiny.
 #[wasm_bindgen]
-impl NodeManager {
-    /// Creates a new [NodeManager] with the given parameters.
+impl MutinyManager {
```

Comment:
Comment above on line 33 still has `NodeManager`, just double check for all of the comments.

Reply:
could catch, didn't find anymore

Reply:
nvm found a test that i renamed